lib/govuk_navigation_helpers/related_items.rb

```diff
@@ -32,6 +36,26 @@ def related_items
 
     attr_reader :content_item
 
+    def register_to_vote_section
+      return if register_to_vote_deadline_reached?
```

Comment:
(It's a matter of taste, but I personally  prefer grouping all of this logic into a small method like `should_show_register_to_vote_section?`)